src/components/Wizard/CreateVmWizard/redux/validations/utils.js

```diff
@@ -0,0 +1,43 @@
+import { get } from '../../../../../selectors';
+
+export const asGenericFieldValidator = (customValidator, getFieldTitle) => (key, field, fields, props) => {
+  const fieldValue = get(field, 'value');
+  let validation = customValidator && customValidator(fieldValue, fields, props);
+
+  // next step is disabled by isValid so empty errors do not need to be shown
+  if (!fieldValue && get(validation, 'isEmptyError')) {
+    validation = null;
+  }
+
+  if (validation) {
+    const title = getFieldTitle(key);
+    if (title) {
+      validation.message = `${getFieldTitle(key)} ${validation.message}`;
+    }
+  }
+
+  return validation === undefined ? null : validation;
+};
+
+export const geValidationUpdate = (config, options, fields, compareField) => {
```

Comment:
I think the 't' in 'get' is missing in the function name.

Reply:
fixed